python/setup.py

```diff
@@ -1,9 +1,63 @@
-from setuptools import setup
-
-setup(name='librmm_cffi',
-      version="0.10.0",
-      packages=["librmm_cffi"],
-      setup_requires=["cffi>=1.0.0"],
-      cffi_modules=["librmm_cffi/librmm_build.py:ffibuilder"],
-      install_requires=["cffi>=1.0.0"],
-      )
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+import os
+import sysconfig
+from distutils.sysconfig import get_python_lib
+
+import versioneer
+from Cython.Build import cythonize
+from setuptools import find_packages, setup
+from setuptools.extension import Extension
+
+install_requires = ["numba", "cython"]
+cython_files = ["rmm/_lib/**/*.pyx"]
+
+if os.environ.get("CUDA_HOME", False):
+    CUDA_HOME = (
+        os.popen('echo "$(dirname $(dirname $(which nvcc)))"').read().strip()
+    )
+cuda_include_dir = os.path.join(CUDA_HOME, "include")
```

Comment:
Unsure if this is the best way to handle this, but it works nicely!

Reply:
A little more logic gives a path in case there is no `CUDA_HOME`: 

```cuda_include_dir = '/usr/local/cuda/include'
cuda_lib_dir = "/usr/local/cuda/lib"

if os.environ.get('CUDA_HOME', False):
    cuda_lib_dir = os.path.join(os.environ.get('CUDA_HOME'), 'lib64')
    cuda_include_dir = os.path.join(os.environ.get('CUDA_HOME'), 'include')```

Reply:
I have that already, but instead of depending on a hardcoded path it queries `which nvcc` and traverses up to get the relevant include directory. In case users (like me) don't use `/usr/local/cuda` as their in use CUDA installation.